packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/mfaoob/use-flow.md

```diff
@@ -0,0 +1,145 @@
+The following sections outline the requests required to implement the MFA OOB flow using direct calls to the Okta OpenID Connect & OAuth 2.0 API.
+
+### Request for tokens
+
+Before you can begin this flow, collect the username and password from the user in a manner of your choosing. Then, make an API call to the Okta [authorization server](/docs/concepts/auth-servers/) `/token` endpoint using the Resource Owner Password grant type. Your request should look something like this:
+
+```bash
+curl --request POST \
+  --url https://${yourOktaDomain}/oauth2/v1/token \
+  --header 'accept: application/json' \
+  --header 'content-type: application/x-www-form-urlencoded' \
+  --data 'client_id=${client_id}&scope=openid profile&grant_type=password&username=${testuser%40example.com}&password={$userpassword}'
+```
+
+Note the parameters that are passed:
+
+- `client_id` matches the client ID of the application that you created in the [Set up your app](#set-up-your-app) section. You can find it at the top of your app's **General** tab.
+- `scope` must be at least `openid`. See the **Create Scopes** section of the [Create an authorization server guide](/docs/guides/customize-authz-server/main/#create-scopes).
+- `grant_type` is `password`, indicating that you're using the Resource Owner Password grant type.
+- `username` is the identifier for the user (email).
+- `password` is the password of the matching user.
+
+Not used in this example:
+
+`grant_types_supported` (Optional) is a list of grant types that the client supports. Some clients aren't able to support all grant types. Use this parameter to specify to Okta that the included grant types are all that the client supports. This allows Okta to return an error if the provided grant types don't satisfy the access policy.
+
+For more information on these parameters, see the `/token` [endpoint](https://developer.okta.com/docs/api/openapi/okta-oauth/oauth/tag/OrgAS/#tag/OrgAS/operation/token).
+
+**Response**
+
+Since this is a two factor flow, Okta sends an HTTP 403 error and includes the `mfa_token` in the response. The `mfa_token` is a unique token used for identifying multifactor authentication flows to link the request to the original authentication flow.
```

Comment:
```suggestion
Since this is a two-factor flow, Okta sends an HTTP 403 error and includes the `mfa_token` value in the response. The `mfa_token` is a unique token used for identifying multifactor authentication flows to link the request to the original authentication flow.
```